Yasgui/packages/yasr/src/plugins/table/index.ts

```diff
@@ -209,6 +209,7 @@ export default class Table implements Plugin<PluginConfig> {
       pageLength: -1,
       data: rows,
       columns: columns,
```

Comment:
can you add a comment why we set this property

Reply:
Ok I will add it, in next MR